packages/core/src/components/modal/modal.scss

```diff
@@ -7,100 +7,95 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-@import '~bootstrap/scss/_functions';
-@import '~bootstrap/scss/mixins/_breakpoints.scss';
-@import '~bootstrap/scss/_variables.scss';
+::backdrop {
+  --ix-dialog-backdrop: var(--theme-color-lightbox, #0000008c);
+}
 
-@import 'common-variables';
+:focus-visible {
+  outline: none !important;
+}
```

Comment:
Have to be imported because at least chrome tries to overwrite it all the time. Have to look in detail to it!